test/benchmarks/perftest.pl

```diff
@@ -491,14 +492,16 @@ ()
     return %result;
 }
 
-sub runtest($$)
+sub runtest
 {
     my $testcasename = shift;
     my $variant = shift;
 
     my %results = ("time", -1, "score", -1);
 
-    $other_switches .= " " . @testlistSwitches{$testcasename};
+    #TODO investigate what was meant to go here, other than adding whitespace on the end
+    #This is a hash slice syntax on an array which is empty, so makes no sense at all
+    #$other_switches .= " " . @testlistSwitches{$testcasename};
```

Comment:
This is most likely obsolete code. I would be fine with getting rid of this line and the variable altogether, and adding the space if it is actually needed.